src/open_inwoner/ssd/tests/test_xml_parsing.py

```diff
@@ -8,7 +8,8 @@
 
 class XMLParsingTests(TestCase):
     def test_uitkering_response_parsing(self):
-        uitkering_file = str(FILES_DIR / "uitkering_response.xml")
+        uitkering_file = str(FILES_DIR / "uitkering1.xml")
+        # uitkering_file = str(FILES_DIR / "uitkering2.xml")
```

Comment:
Lingering comment, and I think you won't need the `str()` cast but keep it a Path?